recipes/simple-build-system/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.0.18" %}
+{% set sha_value = "f8135874cfb599f449363ce11ca45c691bbc7ea4407511c90367cc117dff9f96" %}
+
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: simple-build-system
+  version: {{ version }}
+
+source:
+  url: https://github.com/mctools/simplebuild/archive/v{{ version }}.tar.gz
+  sha256: {{ sha_value }}
+
+build:
+  skip: True  # [win or py<39]
+  number: 0
+  script: python -m pip install --no-deps -vv .
+  entry_points:
+    - sb = _simple_build_system._cli:main
+    - unwrapped_simplebuild = _simple_build_system._cli:unwrapped_main
+    - sbenv = _simple_build_system._cli:sbenv_main
+    - sbrun = _simple_build_system._cli:sbrun_main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tomli  # [py<311]
+    - pybind11 >=2.10.4
+    - cmake >=3.24.2
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - numpy
+    - bash
```

Comment:
`bash` is mostly given by the system and having this a dependency led to problems with some packages on `osx` for me. I would prefer to omit this.
```suggestion
```

Reply:
Ok, I removed it. I hope real bash is always available on both osx and ubuntu then.